### PR TITLE
Handling paths to files in older airflow versions and handling of pendulum versions

### PR DESCRIPTION
**Pull request by walter-hernandez**
Changes made:

- Added handling of paths for LoggingMixin, which changes from Airflow 1.10.11 onwards from:

`from airflow.utils.log.logging_mixin import LoggingMixin`

to

`from airflow import LoggingMixin`

- Also, added handling of importing of Pendulum from the change in Pendulum 1.x to Pendulum 2.x where pendulum.Pendulum was renamed to Pendulum.DateTime

### CODE REVIEW

File: integrations/airflow/marquez_airflow/utils.py [hunk 1]
```diff
@@ -19,7 +19,13 @@
 from airflow.models import Connection
 from airflow.utils.db import provide_session
 from airflow.version import version as AIRFLOW_VERSION
-from pendulum import Pendulum
+
+try:
+    # Import from pendulum 1.x version
+    from pendulum import Pendulum
+except:
```
Comment (wslulciuc) [inline integrations/airflow/marquez_airflow/utils.py:26]: Looks like using a bare **except** fails the CI build [#7010](https://app.circleci.com/pipelines/github/MarquezProject/marquez/839/workflows/8964ba0d-9b54-4384-a673-50dee5f8b1dd/jobs/7010/parallel-runs/0/steps/0-103). Mind specifying the exception? (see Flake8 [E722](https://www.flake8rules.com/rules/E722.html)).

**Review by wslulciuc — CHANGES_REQUESTED** [review]
This looks great, @walter-hernandez! Mind [signing your request](https://github.com/MarquezProject/marquez/blob/main/CONTRIBUTING.md#sign-your-work) and updating your try/except so we can merge your fix?

**Comment by walter-hernandez** [comment]
hello @wslulciuc! Updated the code to indicate the type of exception and included the signing of the commits

**Comment by wslulciuc** [comment]
Thanks, @walter-hernandez!

**Pull request merged**